src/parser/regexp.bnf

```diff
@@ -595,6 +630,9 @@ SourceCharacter
   | ESC_CHAR
     { $$ = Char($1.slice(1), 'simple', @$); $$.escaped = true; }
 
+  | U_CODE_SURROGATE
+    { $$ = Char($1, 'unicode', @$); $$.surrogate_pair = true; }
```

Comment:
let's use JS "camelCase" notation: `isSurrogatePair`

Also, need to update main REAMDE for the `Char` specification for this new field

Reply:
Ok, PR updated!